ci/build.sh

```diff
@@ -35,17 +39,32 @@ fi
 # with an invalid option error.
 export TASTY_NO_CREATE=true
 
+echo "::endgroup::"
+
+echo "::group::Install dependencies"
+
 # Install snapshot dependencies (since these will be cached globally and thus
 # can be reused during the sdist build step)
 $STACK build --only-snapshot $STACK_OPTS
 
+echo "::endgroup::"
+
+echo "::group::Build source distributions"
```

Comment:
I would say the fact that we run the build in a source distribution is an implementation detail which doesn't need to be drawn attention to here. I'd suggest just "Build" or "Build the compiler" instead.

Reply:
Oh sorry, never mind, this part is only creating source distributions, it's not compiling anything. In that case I'd suggest "Create source distributions" since "build" can give people the wrong impression.